translations/en/LC_MESSAGES/messages.po

```diff
@@ -70,40 +70,40 @@ msgid "Missing Colon Error"
 msgstr "We detected that `{command}` needs a `:` at the end of line {line_number}. Starting from level 17, can you start adding a `:` at the end of line with commands?"
 
 msgid "Missing Command"
-msgstr "We detected that the code seems to be missing a command on line {line_number}. Can you try looking at the exercise section to find which command to use?"
+msgstr "We detected that the code seems to be missing a command on line {line_number}. Can you try looking at the adventure section to find which command to use?"
 
 msgid "Missing Inner Command"
-msgstr "We detected that the code seems to be missing a command for the `{command}` statement on line {line_number}. Can you try looking at the exercise section to find which command to use?"
+msgstr "We detected that the code seems to be missing a command for the `{command}` on line {line_number}. Can you try looking at the adventure section to find which command to use?"
 
 msgid "Missing Square Brackets"
-msgstr "We detected that the list you created on line {line_number} is missing square brackets `[]`. Can you try adding square brackets around `[]` the list?"
+msgstr "We detected that the list you created on line {line_number} seems to be missing square brackets `[]`. Can you try adding square brackets around `[]` the list?"
 
 msgid "Missing Variable"
-msgstr "We detected that `{command}` is missing a variable at the start of the line. Can you try writing the variable before the `{command}?`"
+msgstr "We detected that `{command}` seems to be missing a variable at the start of the line. Can you try writing the variable before the `{command}?`"
 
 msgid "Misspelled At Command"
-msgstr "We detected that `{invalid_argument}` seems to have a spelling mistake on line {line_number}. Can you try writing `{command}` instead."
+msgstr "We detected that `{invalid_argument}` seems to have a spelling error on line {line_number}. Can you try writing `{command}` instead."
 
 msgid "No Indentation"
 msgstr "We detected that there seems to be too few {leading_spaces} spaces used on line {line_number}. Can you try increasing indentation by {indent_size} for each new block."
 
 msgid "Non Decimal Variable"
-msgstr "We detected that at line {line_number} you used a number Hedy does not support! Can you try a decimal number, like 2?"
+msgstr "We detected that a number is being used which Hedy does not support, on line {line_number}! Can you try a decimal number, like 2?"
 
 msgid "Parse"
-msgstr "We detected that `{character_found}` is being used on line {location[0]} which is not allowed. Can you try looking for a missing or an extra character on your code?"
+msgstr "We detected that `{character_found}` is being used on line {location[0]} which is not possible. Can you try looking for a missing or an extra character on your code?"
```

Comment:
well it is possible, because the kid did it :) 

Maybe we can say: which hedy can't process or something like that?